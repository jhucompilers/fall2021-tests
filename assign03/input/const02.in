PROGRAM const02;
  CONST N = 20;
  -- a named constant should be allowed as an array size
  VAR arr : ARRAY N OF INTEGER;
  VAR i : INTEGER;

BEGIN
  i := 0;
  WHILE i < N DO
    READ arr[i];
  END;

END.
