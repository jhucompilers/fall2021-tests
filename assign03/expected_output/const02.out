0,CONST,N,INTEGER
0,VAR,arr,ARRAY 20 OF INTEGER
0,VAR,i,INTEGER
